Development/ConfigFiles/ACTL.json

```diff
@@ -6,4 +6,5 @@
 "tokenScanAliases":["^3D bs ASL Mo$","M0","^3D bs ASL turboz$","ASL4D","^T1W_3d$","T1","PhaseImageASL","Phase4D"],
 "tokenVisitAliases":[],
 "tokenSessionAliases":["",""],
-"bMatchDirectories":true}
\ No newline at end of file
+"bMatchDirectories":true,
+"TempRoot":""}
```

Comment:
Hmmm not sure I like this TempRoot strategy, why is it needed? This will confuse the users and is not required right, this is only a dummy step in between for ourselves which we should get rid of

Reply:
We had the AnalysisRoot which is the TempRoot now. I'm not sure if we can remove it everywhere... I'd have to ask @jan-petr about that. I don't see any benefit in removing it right now though, we just risk creating some not predictable bugs...

Reply:
The TempRoot is something that is created internally for the new Import, but it is true that we need it for backwards compatibility with the old configs, so its safer to define here. Should we also check if this is missing and then recreate it based on AnalysisRoot? What do you think @MichaelStritt

Reply:
Ehhm... so for all the files in `ExploreASL\Development\ConfigFiles` we should have both now, so that should be really safe. For everything else I'd just use TempRoot from now on. Don't think we need to implement even more backwards compatibility here.

Reply:
We can remove the AnalysisRoot from all of the files if that's what you want. Removing the TempRoot is only possible if you add some kind of backwards compatibility in the reader.